### PR TITLE
[Notebook] Save snapshot dropdown should be available from "view large" overlay #2922

### PR DESCRIPTION
**Pull request by nikhilmandlik**
Resolves: #2922

Screen Shot 2020-04-13 at 4 32 55 PM
Screen Shot 2020-04-13 at 4 33 04 PM

### CODE REVIEW

**Comment by charlesh88** [comment]
Additional fixes pushed to this branch, significant improvements to Snapshot styling;
  - Added `removeContainer: false` to ExportImageService.js to ease future testing;
  - Hide browse-bar when snapping from view-large overlay;
  - Apply theme background and foreground colors to `s-status-taking-snapshot` element to prevent html2canvas from rendering a white background;
  - Remove background color from `c-so-view` layout frame elements to allow better snapshots in Display and Flex layouts and add slightly translucent look to `c-so-view` layout frame elements when editing;
  - Hide unwanted elements when snapping tables;
- Fixed Overlay and Stacked plots to use consistent flex layout and CSS
classes;
- Moved legacy CSS and cleaned up;
- More hiding and cleanup for better snapshots;

### Author Checklist
1. Changes address original issue? Y
1. Unit tests included and/or updated with changes? N/A
1. Command line build passes? Y
1. Changes have been smoke-tested? Y

File: src/ui/preview/Preview.vue [hunk 1]
```diff
@@ -21,55 +21,42 @@
...
-        ContextMenuDropDown,
-        ViewSwitcher
+        PreviewHeader
     },
     inject: [
         'openmct',
         'objectPath'
     ],
+    props: {
+        notebookEnabled: {
```
Comment (akhenry) [inline src/ui/preview/Preview.vue:48]: Is this used?

Comment (akhenry) [reply src/ui/preview/Preview.vue:48]: @nikhilmandlik Just need to address this to get it merged into Master

Comment (nikhilmandlik) [reply src/ui/preview/Preview.vue]: removed.

**Comment by nikhilmandlik** [comment]
@akhenry , @charlesh88 
This also fixes **[Notebook] Embed links aren't navigating** #2979

**Comment by akhenry** [comment]
## Reviewer Checklist

* Changes appear to address issue? Y
* Appropriate unit tests included? N/A followup issue to address notebook testing
* Code style and in-line documentation are appropriate? Y
* Commit messages meet standards? Y

**Comment by nikhilmandlik** [comment]
Testing Instructions:

1. Create a 'Display layout'
1. Add any existing plot (eg: Overlay Plot) into Display layout
1. Click on 'view large' icon on top-right corner of Overlay Plot.
1. View large should have 'notebook' dropdown.

**Pull request merged**